src/xb-opcode-private.h

```diff
@@ -12,14 +12,17 @@
 
 G_BEGIN_DECLS
 
+#define XB_OPCODE_TOKEN_MAX		32
```

Comment:
Could do with a comment mentioning what it’s for and why the value was chosen as 32.

Reply:
Fixed.